api/scripts/timeLogging.js

```diff
@@ -3,11 +3,12 @@ const db = require('../models')
 module.exports = (() => {
 
     const matchTimeEntry = async (timeEntry) => {
-        return db.models.TimeEntry.findOne({
+        const match = await db.models.TimeEntry.findOne({
             where: {
                 toggl_id: timeEntry.id
             }
         })
+        return match
```

Comment:
@otech47 are these not the same?
`
return db.models.TimeEntry.findOne({...})
`
and
```
const match = await db.models.TimeEntry.findOne({...})
return match
```
Based on this https://jakearchibald.com/2017/await-vs-return-vs-return-await/ should be the same, but I was making some tests on this sandbox https://codesandbox.io/s/priceless-andras-4de81?file=/src/index.js and I'm not getting the expected result

Reply:
As far as I know they are the same, I changed just to check the returned value con a log before the return statement. I will rollback that change. @sofiarm21